test/package/JSforceTestSuite/package.xml

```diff
@@ -61,5 +61,13 @@
         <members>*</members>
         <name>Workflow</name>
     </types>
-    <version>34.0</version>
-</Package>
\ No newline at end of file
+    <types>
+        <members>Admin</members>
+        <name>Profile</name>
+    </types>
+    <types>
+        <members>OauthOidc</members>
+        <name>Settings</name>
+    </types>
+    <version>58.0</version>
```

Comment:
include profile and Oauth settings to enable oauth username-password flow.
This is needed because:
1) new orgs after Summer 23 release have this oauth flow disabled by default
2) test connection uses username-password flow:
https://github.com/jsforce/jsforce/blob/796d828fb4f6905f332bf0752fd66143f220e570/test/helper/connection-manager.ts#L39
**NOTE**: we talked about switching this to use the jwt flow but these tests files are also loaded in browser and `jsonwebtoken` is a node-only pkg.
3) need to set IP range 0.0.0.0 to 255.255.255.255 to allow using a only the password of the scratch org without the security token.